remoting/subsystem/src/main/java/org/jboss/as/remoting/RemotingSubsystem12Parser.java

```diff
@@ -83,10 +83,12 @@ void parseConnector(final XMLExtendedStreamReader reader, final ModelNode addres
                 }
                 case SERVER_NAME: {
                     ConnectorCommon.SERVER_NAME.parseAndSetParameter(value, connector, reader);
+                    serverName = value;
```

Comment:
In this case as we have the parseAndSetParameter calls it looks like these variables should be removed entirely and the relevant if sections removed from below.

Reply:
@darranl I have changed it according to your recommendation.